users/serializers.py

```
@@ -62,4 +64,16 @@ def validate(self, data):
+
+# --- 즐겨찾기 시리얼라이저 ---
+class FavoriteSerializer(serializers.ModelSerializer):
+    # 읽기 전용 필드들 (응답용)
+    favoriteId = serializers.IntegerField(source='favorite_id', read_only=True)
+    companyId = serializers.CharField(source='company.stock_code', read_only=True)
+    companyName = serializers.CharField(source='company.company_name', read_only=True)
+    logoUrl = serializers.URLField(source='company.logo_url', read_only=True)
+
+    class Meta:
+        model = Favorite
+        fields = ['favoriteId', 'companyId', 'companyName', 'logoUrl']
```

Ruff (0.14.11)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)